src/main/java/it/pagopa/ecommerce/commons/utils/RedirectConfigurationKeysConfig.java

```diff
@@ -0,0 +1,39 @@
+package it.pagopa.ecommerce.commons.utils;
+
+import java.net.URI;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+public class RedirectConfigurationKeysConfig {
+
+    private final Map<String, URI> redirectBeApiCallUriMap;
+
+    public RedirectConfigurationKeysConfig(Map<String, URI> redirectBeApiCallUriMap) {
+        this.redirectBeApiCallUriMap = redirectBeApiCallUriMap;
```

Comment:
```suggestion
        this.redirectBeApiCallUriMap =  Collections.unmodifiableMap(redirectBeApiCallUriMap);
```
Wrap input map into an unmodifiable one's since it can be returned by below getter methods and we don't want it to be externally modifiable.

Reply:
Done.